syntax = "proto3";

import "google/protobuf/timestamp.proto";

package heph.telemetry.proto;

message MetricEntry {
    string component = 1;
    string tag = 2;
    google.protobuf.Timestamp log_timestamp = 3;
    string json_values = 4;
}
